webpack.config.production.js

```diff
@@ -1,46 +1,55 @@
+const path = require("path");
 const webpack = require("webpack");
 const { CleanWebpackPlugin } = require("clean-webpack-plugin");
-const HtmlWebpackPlugin = require("html-webpack-plugin");
 const { WebpackManifestPlugin } = require("webpack-manifest-plugin");
 const ZipPlugin = require("zip-webpack-plugin");
 
 const devWebpackConfig = require("./webpack.config");
 
-const isPrerendering = process.env.SCRIVITO_PRERENDER;
-
 function webpackConfig(env = {}) {
+  const { isPrerendering } = env;
+
   const {
     mode: _devMode,
     target: _devTarget,
     devServer: _devServer,
     entry: devEntry,
+    output: devOutput,
     plugins: devPlugins,
     ...sharedConfig
   } = devWebpackConfig({ ...env, production: true });
 
   const plugins = [
     new CleanWebpackPlugin(),
     ...filterDevPlugins(devPlugins),
-    new ZipPlugin({ filename: "build.zip", path: "../", pathPrefix: "build/" }),
+    new ZipPlugin({
+      filename: "build.zip",
+      path: "../",
+      pathPrefix: "build/",
+      exclude: "asset-manifest.json",
+    }),
```

Comment:
Now that I see this: I think we don't need `ZipPlugin` while we are currently prerendering. But I don't know if this complicates things or not.

Reply:
I tried. In the final commit yes, it complicates things. So I left it as is for now.